src/cli/runner.ts

```diff
@@ -6,11 +6,15 @@ import type { Task } from "../index.js";
 import type { D } from "./utils.js";
 
 export type RunnerD = Pick<D, "log" | "error" | "prettyMilliseconds">;
+export interface FailedTask {
+    name: string;
+    error: unknown;
+}
 
 export class Runner {
     private readonly _addedTasks = new Map<Task, Promise<void>>();
 
-    private _errored = false;
+    readonly failedTasks: FailedTask[] = [];
```

Comment:
This can now be `string[]`, since the errors are unused.

Reply:
Good catch, realized I was sorting on the failedTasks array